dask/dataframe/io/io.py

```diff
@@ -181,15 +192,23 @@ def from_pandas(data, npartitions=None, chunksize=None, sort=True, name=None):
     if not has_parallel_type(data):
         raise TypeError("Input must be a pandas DataFrame or Series")
 
-    if (npartitions is None) == (chunksize is None):
-        raise ValueError("Exactly one of npartitions and chunksize must be specified.")
+    # Infer npartitions if npartions and chunksize is None
+    if (npartitions is None) and (chunksize is None):
+        default = config.get("array.chunk-size")
```

Comment:
I'm not sure that we should use `array.chunk-size` here.  Can we move this to an issue for discussion instead?  I would prefer to see a global `chunk-size` or a `dataframe.chunk-size`, but there should probably be discussion on that before we make a decision.

Reply:
I think this is still outstanding.

FWIW, I'm OK with using `array.chunksize` now, and possibly migrating our config to have a top-level `chunksize` and per-collection `chunksize`s that refer to it unless overridden.